Rakefile

```diff
@@ -93,6 +93,11 @@ task :check_deprecations do
   end
 end
 
+desc "Install rubygems to local system"
+task :install => :package do
+  sh "gem install pkg/rubygems-update-#{v}.gem && update_rubygems"
```

Comment:
Should we use our own executable instead of the system executable?

Reply:
@bronzdoc I did some work in https://github.com/rubygems/rubygems/compare/move_gem_to_exe to make it easier to always use the local version of the gem CLI for development tasks, but for now I would like to keep it consistent with the rest of the `Rakefile`, where system rubygems is used for all tasks.

Reply:
Fair 👍